CommunityFridgeMapApi/functions/fridge_reports/app.py

```diff
@@ -0,0 +1,56 @@
+import os
+import logging
+import json
+
+try:
+    from db import get_ddb_connection, FridgeReport
+except:
+    # If it gets here it's because we are performing a unit test. It's a common error when using lambda layers
+    # Here is an example of someone having a similar issue:
+    # https://stackoverflow.com/questions/69592094/pytest-failing-in-aws-sam-project-due-to-modulenotfounderror
+    from dependencies.python.db import get_ddb_connection, FridgeReport
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class FridgReportHandler:
+    def __init__():
+        pass
+
+    @staticmethod
+    def get_fridge_id(event: dict):
+        """
+        Gets the fridge_id from the event dictionary
+        """
+        pathParameters = event.get("pathParameters", {})
```

Comment:
nit: Let's just use chaining here to avoid creating an extra variable?
Like `fridge_id = event.get("pathParameters", {}).get("fridge_id", None)`

Reply:
done: 465bfc801cb533d9b1e3f17f7da73a7cb5af1ce7